src/main/scala/com/azavea/modellab/Service.scala

```diff
@@ -134,7 +177,8 @@ object Service extends SimpleRoutingApp with DataHubCatalog with Instrumented wi
       } ~
       pingPong ~
       pathPrefix("layers"){registerLayerRoute} ~
-      pathPrefix("breaks"){registerColorBreaksRoute}
+      pathPrefix("intbreaks"){registerIntColorBreaksRoute} ~
+      pathPrefix("floatbreaks"){registerFloatColorBreaksRoute}
```

Comment:
Since the data is getting POSTed, option 2 probably is off the table (what with no query params being available). Do you have a second favorite option, @caseypt?